.github/workflows/build.yml

```diff
@@ -0,0 +1,90 @@
+name: Build
+
+on:
+  push:
+    tags:
+      - '*.*.*'
+
+jobs:
+
+  build:
+    name: Build
+    runs-on: ubuntu-latest
```

Comment:
Any chance we could change this build to a Red-Hat flavor OS distribution?

Reply:
why? really, I need to know why? The WMCore is python based code, as such it can run on anything where Python runs. What are so specific about RedHat that WMCore needs? I find much easier to use Debian based distributions than RedHat.

Reply:
somehow my previous comment was duplicated and then wiped out. Let me repeat. 

Why do you need RD Linux? What is special about it for WMCore use-case? In other words, does it required to run WMCore services? The choice of Linux OS is not trivial, and I do understand that at CERN we use CC distribution due to many reason (which are irrelevant here). What is relevant does WMCore require specific Linux OS? If so, which one? How do you foresee to move with its upgrades, etc.? In my experience working with Debian based distros is much more pleasant and easier to maintain, then with RD ones. They don't use RPMs, and rely on more flexible `apt` based packaging. Since WMCore services are pure python ones, it should not be matter which Linux OS we need to use for it.

Reply:
I think it's extremely important to keep the dev/test environment as similar as to those used to run services in production. Given that both VMs and kubernetes clusters are based on Red Hat distribution (Cent OS / Fedora), it's important to test services with the same OS, when possible.

It's all Linux, yes, but still there are differences of the base packages made available, their versions, and even what is available in each repository (as you could see while working with the requirements_py3.txt file).

Reply:
@amaltaro in k8s the nodes are CC, but what you run within them are docker images which can be anything. My question remains, what exactly WMCore services users from RedHat/CC/Fedora? All dependencies are python packages, and unless you give me specific example, I don't see any requirements to run service on specific Linux OS. For instance, I can run WMCore service on macOS and the only question is if service requires access to DB which require DB specific libs. As such what matters if ORACLE provides the libs for given Linux OS.

Reply:
I could be wrong, but it looks like most of our docker images have libraries other than python packages. Here is just one example:
https://github.com/dmwm/CMSKubernetes/blob/master/docker/cmsweb-base/Dockerfile#L8-L11
but cmsweb-base image also has plenty of non-python packages installed.

Reply:
@amaltaro , as I stated already, WMCore is python based application, as such in my view it does not and should not depend on OS. Docker allows multi-stage builds where I can borrow some stuff from one image and put it into another. As such I can use certs from cmsweb and copy them into my final image.

Reply:
@klannon I totally agree, but if we do I rather prefer to have full meeting slot dedicated to this discussion, and not last X minutes. There are so many technical details that it is worth time to discuss them properly. I suggest to start meeting with this topic and if time left go to the usual business with news, etc.